saleor/account/models.py

```diff
@@ -198,3 +202,26 @@ class CustomerNote(models.Model):
 
     class Meta:
         ordering = ("date",)
+
+
+class CustomerEvent(models.Model):
+    """Model used to store events that happened during the customer lifecycle."""
+
+    date = models.DateTimeField(default=now, editable=False)
+    type = models.CharField(
```

Comment:
Maybe we could figure out a different name than built-in function name?

Reply:
We could refactor it to `event_type` but then we would lose consistency with `OrderEvent`. I believe it's fine to define `type` as column name here as it doesn't shadow with the builtin `type` but instead, is only caused an issue with GitHub's Python syntax highlight.